DOCS-3220: Update Unlocking Replicated Dataset (#1313)
truenas/documentation: content/_includes/ReplicatedEncryptedUnlock.md

> 4. Click *Save*.

Reviewer: ```suggestion
4. Click **Save**.
```

{{< /tab >}}
{{< tab "Method 3: Replicate Key Encrypted Dataset/zvol" >}}

> 1. Go to **Storage -> pool/root dataset** on the replication system. Click <i class="fa fa-ellipsis-v" aria-hidden="true" title="Options"></i>  and select **Export Key**.

Reviewer: ```suggestion
1. Go to **Storage -> pool/root dataset** on the replication system. Click <i class="fa fa-ellipsis-v" aria-hidden="true" title="Options"></i> and select **Export Key**.
```

> 2. Either download the Key file, open the file, and change the poolname/dataset to the recieving poolname/dataset or copy the Key code provided in the Key window.

Reviewer: this one reads awkwardly. It sounds like the action is to apply a key file or code to a dataset. If that is the case...
```suggestion
2. Apply the key file or key code to the dataset. You can either download the key file, open that file and change the *pool name/dataset* to the receiving*pool name/dataset*, or copy the key code provided in the **Key** window.
```

> 3. On the receiving pool/dataset: Click <i class="fa fa-ellipsis-v" aria-hidden="true" title="Options"></i>  next to pool/dataset and select *Unlock*.

Reviewer: ```suggestion
3. On the receiving pool/dataset: Click <i class="fa fa-ellipsis-v" aria-hidden="true" title="Options"></i> next to pool/dataset and select **Unlock**.
```

4. There are two ways to unlock the dataset. Either Unset Unlock with Key file. Paste the Key Code into Dataset Key field. (if there is a space character at the end of the key - delete the space.) or Select the Key file that was downloaded and edited.

> 5. Click *Save*.

Reviewer: ```suggestion
5. Click **Save**.
```

> 5. Click *Continue*.

Reviewer: ```suggestion
5. Click **Continue**.
```
